sympy/simplify/tests/test_cse.py

```diff
@@ -175,17 +180,9 @@ def test_non_commutative_order():
     assert cse(l) == ([(x0, B+C)], [x0, A*x0])
 
 
-def test_issue_10228():
-    assert cse([x*y**2 + x*y]) == ([(x0, x*y)], [x0*y + x0])
-    assert cse([x + y, 2*x + y]) == ([(x0, x + y)], [x0, x + x0])
```

Comment:
Please leave the issue number in place so it's easy to find. And I think this simple Add-sub expression extraction could be left, too. So just the first and second lines of this old test function. The issue should be reopened, too. I don't suppose there is an "opens #10228" syntax, however :-)